### PR TITLE
feat: the array sorting method has been created

### PR DESCRIPTION
**Pull request by Zont1k**

### CODE REVIEW

File: Lesson_10_Algorithm/Program.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        PrintArray(array);
+    }
+
+    private static void SortMathod(int[] array)
+    {
+        for (int i = 0; i < array.Length - 1; i++)
+        {
+            for (int j = i + 1; j < array.Length; j++)
+            {
+                if (array[i] > array[j])
```
Comment (blaze6950) [inline Lesson_10_Algorithm/Program.cs:20]: introduce a variable for `array[i]` and move out it from this loop

Comment (blaze6950) [reply Lesson_10_Algorithm/Program.cs:20]: This will help to save time on getting a value using its index

File: Lesson_10_Algorithm/Program.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                {
+                    int temp = array[i];
```
Comment (blaze6950) [inline Lesson_10_Algorithm/Program.cs:22]: Try to swap values without the additional variable. It isn't so useful, but it is a challenging task for breaking your brain. This kind of task can train your brain and this is beneficial for you.

File: Lesson_10_Algorithm/Program.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    static void Main(string[] args)
+    {
+        int[] array = { 4, 9, 7, 6, 2, 3, 8, 5, 0, 1 };
+
+        SortMathod(array);
+
+        PrintArray(array);
+    }
+
+    private static void SortMathod(int[] array)
```
Comment (blaze6950) [inline Lesson_10_Algorithm/Program.cs:14]: Can you identify the used sort method? What pros & cons it has? What other types of sorting do you know?
Try to create a new method for implementing there another sorting algorithm. Check the execution time. The best option would be to compare the performance using benchmark

**Review by blaze6950 — CHANGES_REQUESTED** [review]

File: Lesson_10_Algorithm/Program.cs (hunk 1 above)
Comment (Zont1k) [reply Lesson_10_Algorithm/Program.cs:20]: done!

Comment (Zont1k) [reply Lesson_10_Algorithm/Program.cs:20]: done!

File: Lesson_10_Algorithm/Program.cs (hunk 3 above)
Comment (Zont1k) [reply Lesson_10_Algorithm/Program.cs:14]: | Method              | Mean      | Error    | StdDev   |
|-------------------- |----------:|---------:|---------:|
| BubbleSortBenchmark |  44.56 ns | 2.081 ns | 5.905 ns |
| QuickSortBenchmark  | 115.20 ns | 2.300 ns | 3.842 ns |

**Review by blaze6950 — APPROVED** [review]

**Pull request merged**